picknik_ament_copyright/copyright_names.py

```diff
@@ -10,7 +10,7 @@
 #      notice, this list of conditions and the following disclaimer in the
 #      documentation and/or other materials provided with the distribution.
 #
-#    * Neither the name of the PickNik Consulting nor the names of its
+#    * Neither the name of the PickNik Inc. nor the names of its
 #      contributors may be used to endorse or promote products derived from
```

Comment:
super nit - Should the line wrapping here be adjusted? Unclear what line length is set to

Reply:
Looks like the ament_copyright templates ([BSD-3, for example](https://github.com/ament/ament_lint/blob/master/ament_copyright/ament_copyright/template/bsd_3clause_header.txt)) don't adjust line wrapping at all to account for the length of the copyright holder's name.

Since ament_copyright looks for exact line-by-line matches to the expected license format, fixing the line wrapping would actually cause it to fail the license check!